pkg/gcr/gcr.go

```diff
@@ -10,22 +10,23 @@ import (
 )
 
 type TrustedGcrRepository struct {
-	ref    name.Reference
-	auth   authn.Authenticator
-	config *trust.Config
+	ref        name.Reference
+	repoAuth   authn.Authenticator
```

Comment:
In Docker's term, a "repository" represent an image with tags, the place to push images is "registry". So I think we may rename "repoAuth" to "registryAuth".